azure/ConsiderationsForServiceDesign.md

```diff
@@ -225,33 +226,111 @@ a _status monitor_, which is an ephemeral resource that will track the status an
 The status monitor resource is distinct from the target resource (if any) and specific to the individual
 operation request.
 
-A POST or DELETE operation returns a `202 Accepted` response with the status monitor in the response body.
-A long-running POST should not be used for resource create -- use PUT as described below.
-PATCH must never be used for long-running operations -- it should be reserved for simple resource updates.
-If a long-running update is required it should be implemented with POST.
+There are four types of LROs allowed in Azure REST APIs:
+
+1. An LRO to create or replace a resource that involves additional long-running processing.
```

Comment:
Minor: I think the list is somewhat redundant as it's pretty much applies to every operation?

Reply:
This list is a categorization of LRO types that each have specific requirements. It is intended to prepare the reader for the description of each type separately below.